### ISSUE TITLE
gas.py has a problem with gas.cleanup() called first.

### ISSUE THREAD

**Issue by dglaude** [pimoroni/EnviroPlus-FeatherWing#7]
Running this simple code:
```
from pimoroni_envirowing import gas

gas.cleanup()

```

Trigger the following output:
> main.py output:
Traceback (most recent call last):
  File "main.py", line 3, in <module>
  File "pimoroni_envirowing/gas.py", line 50, in cleanup
NameError: name 'enable' is not defined

This is actually what the demo code is doing:
```
from pimoroni_envirowing import gas
import time
while True:
    try:
        print(gas.read_all())
        time.sleep(0.1)
    finally:
        gas.cleanup()
```

At least if the first call to gas.read_all() fail with an exception.

**Comment by Gadgetoid** [comment]
Looks like `enable` should be defined in the top of the script as `enable = None`. In fact I'd probably refactor that to `enable_pin`.

Then a check for `if enable_pin is not None:` in `cleanup()` should suffice.

There's always the trap the exception method, too, but in cases like this I definitely prefer the former.

**Comment by Gadgetoid** [comment]
This is- with any luck- fixed in #10
